content/en/providers/check-in/sp/_index.md

```diff
@@ -1163,6 +1163,12 @@ Each level is represented by a URI as follows:
 - **High**: Substantial + multi-factor authn (not yet supported, TBD):
   `https://aai.egi.eu/LoA#High`
 
+Moreover, EGI Check-in follows the [REFEDS Assurance framework](https://wiki.refeds.org/display/ASS/REFEDS+Assurance+Framework+ver+1.0).
+The EGI Check-in conveys any RAF values provided by the IdP directly to the SP,
+through the aforementioned methods. Furthermore, Check-in will append into the User's
+profile any additional LoA, if the user is eligible for it. For example,
+a user having a Verified Email is eligible for the RAF value ```https://refeds.org/assurance/IAP/low```
```

Comment:
Please introduce the RAF acronym, same for LoA. 
It's also better to avoid acronyms when it's not needed as it ease reading.

Reply:
Done

Reply:
Done